packages/api/lib/granules.js

```diff
@@ -125,7 +126,7 @@ async function moveGranuleFilesAndUpdateDatastore(params) {
       });
       // Add updated file to postgresDatabase
     } catch (error) {
-      updatedFiles.push({ bucket: file.bucket, key: file.key, fileName: file.fileName });
+      updatedFiles.push({ bucket: file.bucket, key: file.key, fileName: getNameOfFile(file) });
```

Comment:
I'm adding this method to get the `fileName` from the `key` because `fileName` will not exist if we haven't called `moveGranule`. `moveGranule` would have set it with this same helper. 

I believe the previous logic would always (or at least with the test data we have) set it to `undefined`. 

It's very confusing so I'm open to discussion.

Reply:
No, you're correct, we talked about that a bit on Monday.  The correct value is needed for the update regardless, even if it is in theory vestigal after some of the work in-flight.   We're setting it in moveGranuleFile, the equivalent should be set in the dynamo record. 

Re-reviewing prior work, I'm a little concerned about the lack of transaction around the dynamo update, but I'll go back and see why we thought that was acceptable in #2186 later.